EIPS/eip-5593.md

```diff
@@ -28,9 +28,9 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 The provider objects, e.g. `window.ethereum`, are expected to only inject the Ethereum Provider APIs in secure context when conforming with this standard. The following restrictions are REQUIRED for conformant wallets:
 
 - Provider objects MAY be accessible in private (incognito) windows.
-- The origin MUST be a [potentially trustworthy origin][w3c-secure-context-trustworthy-origin] to have access to `window.ethereum`. This can be checked using `window.isSecureContext`, including inside iframes.
-    - Secure contexts include sites that are served from HTTPS but also HTTP `localhost`. 
-    - The browser implementation MAY also support configured [potentially trustworthy origins] that would normally not be considered trustworthy if the user configures their browser to do so. See the [Development Environments section of Secure Contexts][w3c-secure-context-dev-env] for additional details. For example, in Chromium based browsers this is done via the `chrome://flags/#unsafely-treat-insecure-origin-as-secure` flag.
+- To have access to `window.ethereum`, the origin MUST be a `potentially trustworthy origin` according to the user agent. This can be checked using `window.isSecureContext`, including inside iframes.
```

Comment:
This is a bad normative statement. How is an implementer supposed to know what a "potentially trustworthy origin" is without a definition?